pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.ruff]
+line-length = 120
+# Enable Pyflakes `E` and `F` codes by default.
+select = [
+    "E", "W",  # see: https://pypi.org/project/pycodestyle
+    "F",  # see: https://pypi.org/project/pyflakes
+    "I",  # see: isort
```

Comment:
If we enable ruff's isort feature we should remove isort related hooks and config from pre-commit config